tests/test_misc.py

```diff
@@ -43,7 +46,18 @@ def test_multi_dict():
     assert multi_dict.original_items() == [("b", ["abc"])]
 
 
-def test_get_logger(caplog):
+def test_multi_dict_init_empty() -> None:
+    multi_dict = MultiDict(None)
```

Comment:
If it is a desired feature, maybe the default value for `mapping` can be equal to `None` 😉 

Thanks to that, we don't have to pass it and simple `MultiDict()` will behave the same.